src/steps/account/converter.ts

```diff
@@ -5,21 +5,15 @@ import {
 
 import { Entities } from '../constants';
 
-export function createAccountEntity(): Entity {
+export function createAccountEntity(account: { hostname: string }): Entity {
   return createIntegrationEntity({
     entityData: {
-      source: {
-        id: 'acme-unique-account-id',
-        name: 'Example Co. Acme Account',
-      },
+      source: account,
       assign: {
-        _key: 'acme-unique-account-id',
+        _key: account.hostname,
```

Comment:
```suggestion
        _key: `cloudbees_account:${account.hostname}`
```

Doesn't have to be separate function if it's not going to be used/called more than once.